### PR TITLE
Add production Qwen3.8-27B support

### PR DESCRIPTION
**Pull request by justinchuby**
## Summary

- add production support for pinned `Qwen/Qwen3.8-27B@1d4bf0f2ff6012fd82039f2fa52739d0dd7c60c0` as a checkpoint alias of the existing dense Qwen3.5 hybrid vision-language architecture
- preserve the exact 64-layer 48-DeltaNet/16-full-attention text schedule, 27-layer vision encoder, image/video processor contracts, mixed-batch scatter order, dtype boundaries, and optional separately packaged MTP drafter
- preserve canonical `qwen3_5` registry regression coverage on `Qwen/Qwen3.5-2B`; Qwen3.8 uses its own pinned case/config assertions instead of replacing that mapping
- emit topology-faithful ORT GenAI metadata without a runtime capability gate: decoder-only packages keep `qwen3_5_text`, while multimodal Qwen3.5/Qwen3.8 packages emit `qwen3_5`
- remove the standalone Qwen3.8 Olive example, its coupled reduced-real test module, and its private reduced golden fixtures; core model, processor, metadata, and parity coverage remain unchanged

Stacked on #487 final skill-split head `a8cd77570bca980861eeadab9e6fa077464f0138`. Exact PR head: `59af1a51f1a162e9444a9be1ea5785ca4ec0ef0b`. Tracks #483.

## Validation

- the real processor from the pinned official revision drives deterministic tiny-model HF/ONNX pipeline parity for image-only, video-only, and a mixed two-row batch with opposite placeholder order: image max abs `0.00697723`, video `0.00411959`, mixed `0.00677243`; cosine is above `0.99999` and final argmax matches
- exact raw-config assertions cover the 48/16 hybrid schedule, text/vision dimensions, processor token contracts, and intentional separate MTP packaging
- packed-vision coordinates are emitted once and shared by interpolation, rotary, and frame-boundary consumers; media ownership uses boundary `ScatterElements` + `CumSum` in O(patches + media)
- deterministic Qwen3.5-VL benchmark: `428 -> 467` top-level nodes (`+9.1%`, model size unchanged), below the unchanged 10% blocker threshold
- topology-specific metadata regression coverage generates both package shapes: decoder-only `qwen3_5_text` and multimodal `qwen3_5`
- post-removal YAML/coverage/focused suite: `987 passed, 227 skipped`; diff lintrunner clean; independent review found no stale reduced/Olive claims or dangling references
- canonical non-integration suite before the deletion-only follow-up: `3962 passed, 61 skipped`

## L4/L5 waiver

The pinned official checkpoint is 55.6 GB and exceeds hosted CI storage and GPU memory; no reduced or quantized private fixture is committed. The Qwen3.8 case records this exact `ci_skip_reason` rather than claiming retained executable evidence. Earlier reduced-real and Olive development artifacts were removed and are not part of the final PR validation surface.

## CI

Current head `59af1a51f1a162e9444a9be1ea5785ca4ec0ef0b`:

- Benchmark run `32314325378` is complete and successful (base, head, and comparison).
- CI run `32314325553` is still active. Lint, affected-model detection, L1, L3, and lintrunner have completed successfully; the Linux/Windows test matrices are in progress, while Integration (fast), L4, and L5 are queued.
- Architecture Diff run `32314325423` is in progress.

Historical run `31870831200` belongs to pre-removal head `067cb489599687b74f96574de20133cb96e016a7`, not the final head. It is retained only as historical context: its three red jobs reproduced on the exact #487 base run `31869640946`, but it is not cited as current-head CI evidence.

PR #498 remains ready for maintainer review; auto-merge is off.

### CODE REVIEW

File: src/mobius/models/qwen_vl.py [hunk 1]
```diff
@@ -1022,21 +1026,44 @@ def forward(
...
         image_mask = op.Equal(input_ids, op.Constant(value_int=self.image_token_id))
-        image_mask_3d = op.Unsqueeze(image_mask, [-1])
-        mask_int = op.Cast(image_mask, to=7)  # INT64
-        cumsum = op.CumSum(mask_int, op.Constant(value_int=1))
-        indices = op.Clip(
-            op.Sub(cumsum, op.Constant(value_int=1)),
-            op.Constant(value_int=0),
+        if self.video_token_id is None:
+            video_mask = op.CastLike(False, image_mask)
+        else:
```
[the remark this answers is not in the document]
Comment (justinchuby) [reply src/mobius/models/qwen_vl.py:1035]: Fixed in 54433fd. The absent-video path now builds an all-false ONNX boolean tensor as `Not(Equal(input_ids, input_ids))`, so no Python scalar enters the graph. I also added a runtime graph test with `video_token_id=None` that verifies image feature scatter across two batch rows.

File: src/mobius/models/qwen_vl.py [hunk 2]
```diff
@@ -1022,21 +1026,44 @@ def forward(
...
+                op.CumSum(flat_video_mask, op.Constant(value_int=0)),
+                op.Constant(value_int=1),
+            ),
+            op.ReduceSum(flat_image_mask, keepdims=0),
+        )
+        flat_indices = op.Where(
+            op.CastLike(flat_image_mask, image_mask),
+            image_indices,
+            video_indices,
         )
```
[the remark this answers is not in the document]
Comment (justinchuby) [reply src/mobius/models/qwen_vl.py:1061]: Fixed in 54433fd. The code now flattens the original boolean `image_mask` once and uses it directly as the `Where` condition; the INT64 cast remains separate and is used only for cumulative index arithmetic. Existing mixed image/video runtime coverage still verifies independent image/video streams with opposite placeholder order.
